2024-10-21 21:52:14.677033+08:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS athletes (\000d\000a  id INTEGER PRIMARY KEY [*]AUTOINCREMENT,\000d\000a  name TEXT,\000d\000a  trainingPlan TEXT,\000d\000a  weight DOUBLE,\000d\000a  category TEXT,\000d\000a  competitions INTEGER,\000d\000a  coachingHours INTEGER,\000d\000a)"; expected "HASH, CONSTRAINT, COMMENT, UNIQUE, NOT NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE IF NOT EXISTS athletes (
  id INTEGER PRIMARY KEY AUTOINCREMENT,
  name TEXT,
  trainingPlan TEXT,
  weight DOUBLE,
  category TEXT,
  competitions INTEGER,
  coachingHours INTEGER,
) [42001-232]
2024-10-21 21:52:44.774899+08:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS athletes (\000d\000a  id INTEGER PRIMARY KEY AUTO_INCREMENT,\000d\000a  name TEXT,\000d\000a  trainingPlan TEXT,\000d\000a  weight DOUBLE,\000d\000a  category TEXT,\000d\000a  competitions INTEGER,\000d\000a  coachingHours INTEGER,\000d\000a[*])"; expected "identifier"; SQL statement:
CREATE TABLE IF NOT EXISTS athletes (
  id INTEGER PRIMARY KEY AUTO_INCREMENT,
  name TEXT,
  trainingPlan TEXT,
  weight DOUBLE,
  category TEXT,
  competitions INTEGER,
  coachingHours INTEGER,
) [42001-232]
2024-10-21 22:19:04.242945+08:00 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Benjamin Juaton/Lithan/test.mv.db [2.3.232/7]" [50000-232]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:344)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:682)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at main.Main.main(Main.java:14)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Benjamin Juaton/Lithan/test.mv.db [2.3.232/7]" [50000-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Benjamin Juaton/Lithan/test.mv.db [2.3.232/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 12 more
2024-10-21 22:31:00.180804+08:00 database: wrong user or password; user: "GEO"
org.h2.message.DbException: Wrong user name or password [28000-232]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:344)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:291)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:811)
	at org.h2.server.web.WebApp.login(WebApp.java:1038)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:154)
	at org.h2.server.web.WebThread.run(WebThread.java:103)
	at java.base/java.lang.Thread.run(Thread.java:1575)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:522)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 16 more
2024-10-21 22:31:03.327088+08:00 jdbc[5]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-232]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:217)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:134)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:291)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:811)
	at org.h2.server.web.WebApp.login(WebApp.java:1038)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:154)
	at org.h2.server.web.WebThread.run(WebThread.java:103)
	at java.base/java.lang.Thread.run(Thread.java:1575)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 13 more
2024-10-21 22:31:50.781656+08:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "ATHLETES" already exists; SQL statement:
CREATE TABLE athletes (id NUMERIC, weight DECIMAL) [42101-232]
2024-10-21 22:35:36.077299+08:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "drop table athletes[*]()"; SQL statement:
drop table athletes() [42000-232]
2024-10-21 22:35:47.636996+08:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DROP TABLE athletes[*]()"; SQL statement:
DROP TABLE athletes() [42000-232]
2024-10-21 22:36:13.067735+08:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE athletes ([*]"; expected "identifier"; SQL statement:
CREATE TABLE athletes ( [42001-232]
2024-10-21 22:40:24.409360+08:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE athletes (\000d\000a  id NUMERIC NOT NULL PRIMARY KEY [*]AUTOINCREMENT,\000d\000a  name VARCHAR(255) NOT NULL,\000d\000a  training_plan VARCHAR(255) NOT NULL,\000d\000a  weight DECIMAL NOT NULL,\000d\000a  category VARCHAR(255),\000d\000a  competitions NUMERIC,\000d\000a  coaching_hours NUMERIC\000d\000a)"; expected "HASH, CONSTRAINT, COMMENT, UNIQUE, NOT NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE athletes (
  id NUMERIC NOT NULL PRIMARY KEY AUTOINCREMENT,
  name VARCHAR(255) NOT NULL,
  training_plan VARCHAR(255) NOT NULL,
  weight DECIMAL NOT NULL,
  category VARCHAR(255),
  competitions NUMERIC,
  coaching_hours NUMERIC
) [42001-232]
2024-10-21 22:46:53.007494+08:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Alex" not found; SQL statement:
INSERT INTO athletes (name, training_plan, weight, competitions, coaching_hours)
VALUES ("Alex", "Intermediate", 98.5, 1, 5) [42122-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:299)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:489)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:645)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:561)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1164)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:245)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1344)
	at org.h2.server.web.WebApp.query(WebApp.java:1142)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1108)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1095)
	at org.h2.server.web.WebThread.process(WebThread.java:190)
	at org.h2.server.web.WebThread.run(WebThread.java:103)
	at java.base/java.lang.Thread.run(Thread.java:1575)
2024-10-21 23:08:00.609813+08:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * [*]athletes"; SQL statement:
SELECT * athletes [42000-232]
